src/Behat/Gherkin/Node/TableNode.php

```diff
@@ -52,11 +52,19 @@ public function __construct(array $table)
                 throw new NodeException('Table does not have same number of columns in every row.');
             }
 
+            if (!is_array($row)) {
+                throw new NodeException('Table is not two-dimensional.');
+            }
+
             foreach ($row as $column => $string) {
                 if (!isset($this->maxLineLength[$column])) {
                     $this->maxLineLength[$column] = 0;
                 }
 
+                if (is_array($string)) {
```

Comment:
@stof I tried `!is_string()` first, but a whole bunch of unrelated unit tests failed because they were passing integers. I could change the variable name from `$string` to `$value`, since the assumption isn't necessarily valid.